lib/hal/hal.ex

```diff
@@ -7,7 +7,8 @@ defmodule Inky.HAL do
 
   @callback init(opts :: map()) :: io_state()
   @callback handle_update(
-              opts :: map(),
+              pixels :: map(),
+              border :: :white | :black | :red | :yellow | :accent,
```

Comment:
Are we always accepting `:red | :yellow | :accent` where we accept color? I thought it was white, black, accent only. Should it be?

Reply:
We accept all colors, https://github.com/pappersverk/inky/blob/master/lib/hal/rpihal.ex#L12, and have done for quite some time. 🤷‍♂

Reply:
Ok, cool. But only for very small values of all ;)